src/js/components/DataTable/__tests__/__snapshots__/DataTable-test.js.snap

```diff
@@ -5128,118 +5128,83 @@ exports[`DataTable search 2`] = `
   -webkit-flex-direction: column;
   -ms-flex-direction: column;
   flex-direction: column;
-  -webkit-flex: 0 0 auto;
-  -ms-flex: 0 0 auto;
-  flex: 0 0 auto;
+  -webkit-flex: 1 1;
```

Comment:
These changes points out a regression on how `flex` was calculated, the snapshots files shouldn't change for the refactoring effort (Clean snapshots are an indication of an intact component)

Reply:
will look into it

Reply:
Thanks buddy!

Reply:
I believe the snapshots aren't correct in the master branch, here's what I've found out:

Here's what the search looks like without an input value:

![image](https://user-images.githubusercontent.com/7442967/66668073-c6d2dc00-ec2a-11e9-9e38-a3b6fccf428e.png)

But, with a value, it looks like this:

![image](https://user-images.githubusercontent.com/7442967/66668130-deaa6000-ec2a-11e9-8ea0-cee048386756.png)


Now, if you take a look at the specs, there are two snapshots being taken, one before the input, and one with an input (`src/js/components/DataTable/__tests__/DataTable-test.js:107`):

```
    expect(container.firstChild).toMatchSnapshot();
    fireEvent.click(container.querySelector('[aria-label="focus-search-a"]'));
    const searchInput = container.querySelector('[name="search-a"]');
    >> expect(document.activeElement).toBe(searchInput);
    fireEvent.change(searchInput, {
      target: { value: '[' },
    });
    >> expect(container.firstChild).toMatchSnapshot();
```

However, in the master branch, both snapshots haves the same values for the `flex`attributes (`src/js/components/DataTable/__tests__/__snapshots__/DataTable-test.js.snap:4867 and :5131`):

```
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
```

However, with the refactor, the snapshots are reflecting the visual changes the component goes through.

Reply:
Interesting, thank you for taking a look at that.
I agree the snapshots should look different after using the search input field.
@ericsoderberghp I'd like your look at this snapshot as well.